code-examples/low-latency-sessions/main.go

```diff
@@ -0,0 +1,134 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/gin-gonic/gin"
+	"github.com/go-redis/redis/v8"
+
+	client "github.com/ory/kratos-client-go"
+)
+
+const (
+	// For local development, it's requires to use
+	// ory proxy to make cookies work fine.
+	// Replace it with http://projectId.projects.oryapis.com
+	// for production usage.
+	sdkURL = "http://localhost:4000/.ory"
+	// For local development, it's requires to use
+	// ory proxy to make cookies work fine.
+	// Replace it with http://projectId.projects.oryapis.com/ui/login
```

Comment:
instead of `projectId` please use `<project_slug>` - this form is used in many places in our docs already

Reply:
Done